### PR TITLE
Tutorial Panel added to main menu and two game scenes

### PR DESCRIPTION
**Pull request by WendyFu004**
Question mark on the left top corner is the help bottom used to open the tutorial page, and this button shown in main menu and both game scenes. When the tutorial page is opening, the time scale become zero, so the game is paused during the time when the tutorial is on.

First try:  Click the bottom to open the tutorial page and close the page with click again.

** Now what we have: H key is used to open and close the tutorial now! I found that we could use both arrow   keys and mouse to choose the items so I change to use the key otherwise the mouse click will mess up with spacebar.

**made an simple image for the background. Replaced the original one, and might make a cover (poster?) if time enough.

### CODE REVIEW

File: Assets/Scripts/UI/PauseMenu/PauseMenu.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 public class PauseMenu : MonoBehaviour
 {
-    private static bool GameIsPaused = false;
+    public static bool GameIsPaused = false;
```
Comment (Aether700) [inline Assets/Scripts/UI/PauseMenu/PauseMenu.cs]: Please do not make this variable public and use getters/setters to access/modify the value of the variable

**Review by Aether700 — CHANGES_REQUESTED** [review]
Very good work. There is a coding issue I would like you to fix though. I left a comment describing the issue above.

Also, small tweaks I would suggest: in the race track scene move the help indicator at the very top left corner of the screen and make sure the question mark sprite is not highlighted like a button when you hover it with your mouse because it looks like a button that can be clicked when it is not

**Comment by afourneaux** [comment]
Absolutely gorgeous. I noticed "Rocket" is abbreviated to "Rock" which seems odd to me, but the images make it very clear.

Fantastic work, I love the look of it!

**Comment by afourneaux** [comment]
In the main menu, it looks like the H? icon is blocking clicks to the dumpster. Simply toggling "Raycast Target" on the QuestionMark game object should resolve this

File: Assets/Scripts/Tutor/TutForMenu.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TutForMenu : MonoBehaviour
+{
+    public GameObject TutorialPanel;
```
Comment (Aether700) [inline Assets/Scripts/Tutor/TutForMenu.cs]: Please make this variable private and add the serializeField attribute to it

File: Assets/Scripts/Tutor/Tutoral.cs [hunk 3]
```diff
@@ -5,6 +5,8 @@
 public class Tutoral : MonoBehaviour
 {
     public GameObject TutorialPanel;
```
Comment (Aether700) [inline Assets/Scripts/Tutor/Tutoral.cs]: Again make this variable private and add the serialize field to it

**Review by Aether700 — DISMISSED** [review]
nice work

**Review by Aether700 — CHANGES_REQUESTED** [review]
The canyon scene crashes when loaded Please fix

**Comment by Aether700** [comment]
The issue is probably how you merged main
I would look there to make sure that's not the problem first

**Comment by afourneaux** [comment]
Please see #49 where the merge was corrected

**Pull request closed without merging**